packages/back-end/src/util/organization.util.ts

```diff
@@ -82,13 +152,45 @@ export function getUserPermissions(
     projects: {},
   };
 
+  // Build the user's user-level project permissions
   memberInfo?.projectRoles?.forEach((projectRole: ProjectMemberRole) => {
     userPermissions.projects[projectRole.project] = {
       limitAccessByEnvironment: projectRole.limitAccessByEnvironment || false,
       environments: projectRole.environments || [],
       permissions: roleToPermissionMap(projectRole.role, org),
     };
   });
+
+  // If the member's global role is admin, they already have all permissions
```

Comment:
Is this logic for admin roles in the code? Was looking for something like `memberInfo?.role === 'admin'` but don't see it

Reply:
I did have it in, but ultimately removed it accidentally left the comment in. 

After reviewing the logic - I felt like there might be some corner cases where we wouldn't want to skip building the team permissions, so I removed it.

I'm having trouble recalling the specifics right now, unfortunately.

For now, I've just removed the comment.